mantidproject/mantid
# Improvements for MSlice manual test instructions

**Description of work.**

Updated the instructions for MSlice manual testing.

**To test:**

Follow the test instructions and see if they make sense. Also, please look for typos and grammatical errors.

The associated issue is https://github.com/mantidproject/mantid/pull/592 in the MSlice project: https://github.com/mantidproject/mslice/issues/647 

*This does not require release notes* because **it is an addition to the developer documentation for manual testing**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
